quaint/src/connector/postgres/native/mod.rs

```diff
@@ -292,6 +296,21 @@ impl PostgreSql {
         })
     }
 
+    /// Create a new websocket connection to managed database
+    pub async fn new_with_websocket(url: PostgresWebSocketUrl) -> crate::Result<Self> {
+        let client = connect_via_websocket(url).await?;
+
+        Ok(Self {
+            client: PostgresClient(client),
+            socket_timeout: None,
+            pg_bouncer: false,
+            statement_cache: Mutex::new(StatementCache::new(0)),
+            is_healthy: AtomicBool::new(true),
+            is_cockroachdb: false,
```

Comment:
No more `let is_cockroachdb = conn.parameter("crdb_version").is_some();`, then?

Reply:
This guaranteed to connect only to postgres instances